src/test/java/no/unit/nva/model/PublicationJournalArticleTest.java

```diff
@@ -40,7 +41,7 @@ public void objectMappingOfPublicationClassReturnsSerializedJsonWithJsonLdFrame(
     }
 
     @DisplayName("Test publications can be serialized/deserialized")
-    @ParameterizedTest
+    @ParameterizedTest(name = "Test Publication context Journal with Instance type {0} can be (de-)serialized")
```

Comment:
In this test you should verify that all fields of either the expected object or the actual object are non empty